doc/logging.md

```diff
@@ -0,0 +1,17 @@
+# How does Happy test log work
+
+## Types of Happy logs
+  * Happy debug logs — Happy will run as test coordination and orchestrator, facilitating the activities of the virtual test device nodes. Happy debug logs will have logs from Happy tool only.
+
+  * Happy device logs — Happy will host virtual network which has multiple virtual devices, Happy device logs will have logs for software running on the virtual device, OpenWeave is one of the software example, so device logs will have OpenWeave logs.
```

Comment:
We can reword this too.  As for "software" does that mean anything that can be running on the node?  Any Linux binary visible to the namespace, or just stuff related to Happy commands?

Device logs - Logs related to software running on individual virtual device nodes within a Happy test network.

Reply:
ok

Reply:
Hi Jeff: 
    The "software" here means "openweave", how about change to "software such as openweave"